BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Views/SkontoProceedView.swift

```diff
@@ -0,0 +1,174 @@
+            dividerView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            dividerView.heightAnchor.constraint(equalToConstant: Constants.dividerViewHeight),
+
+            totalLabel.topAnchor.constraint(equalTo: dividerView.bottomAnchor, constant: Constants.padding),
+            totalLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: Constants.padding),
+            totalLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -Constants.padding),
+
+            totalValueLabel.topAnchor.constraint(equalTo: totalLabel.bottomAnchor, constant: 0),
+            totalValueLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: Constants.padding),
+
+            skontoBadgeView.centerYAnchor.constraint(equalTo: totalValueLabel.centerYAnchor),
+            skontoBadgeView.leadingAnchor.constraint(equalTo: totalValueLabel.trailingAnchor,
+                                                     constant: Constants.badgeSpacing),
+            skontoBadgeView.trailingAnchor.constraint(lessThanOrEqualTo: trailingAnchor,
+                                                     constant: -Constants.padding),
+
+            skontoBadgeLabel.topAnchor.constraint(equalTo: skontoBadgeView.topAnchor,
+                                                  constant: Constants.badgeVerticalPadding),
+            skontoBadgeLabel.bottomAnchor.constraint(equalTo: skontoBadgeView.bottomAnchor,
+                                                     constant: -Constants.badgeVerticalPadding),
+            skontoBadgeLabel.leadingAnchor.constraint(equalTo: skontoBadgeView.leadingAnchor,
+                                                      constant: Constants.badgeHorizontalPadding),
+            skontoBadgeLabel.trailingAnchor.constraint(equalTo: skontoBadgeView.trailingAnchor,
+                                                       constant: -Constants.badgeHorizontalPadding),
+
+            payButton.topAnchor.constraint(equalTo: totalValueLabel.bottomAnchor, constant: Constants.verticalPadding),
+            // TODO: no safe area bottom padding in design
+            payButton.bottomAnchor.constraint(equalTo: safeAreaLayoutGuide.bottomAnchor,
+                                              constant: -Constants.verticalPadding),
+            payButton.centerXAnchor.constraint(equalTo: centerXAnchor),
+            payButton.leadingAnchor.constraint(equalTo: leadingAnchor, constant: Constants.padding),
+            payButton.heightAnchor.constraint(greaterThanOrEqualToConstant: Constants.buttonHeight)
+        ])
+    }
+
+    private func bindViewModel() {
+        configure()
+        viewModel.addStateChangeHandler { [weak self] in
+            guard let self else { return }
+            self.configure()
+        }
+    }
+
+    private func configure() {
+        let isSkontoApplied = viewModel.isSkontoApplied
+        self.skontoBadgeView.isHidden = isSkontoApplied ? false : true
```

Comment:
[suggestion] please rewrite this `self.skontoBadgeView.isHidden = !viewModel.isSkontoApplied`